ccmlib/scylla_node.py

```diff
@@ -1246,8 +1203,7 @@ def no_pending_flushes():
                     if pending_flushes_count > 0:
                         pending_flushes = True
             return not pending_flushes
-        result = wait_for(no_pending_flushes, timeout=wait_timeout, step=1.0)
-        if result is None:
```

Comment:
How did result end up as `None` previously?

This looks like a bug fix on its own, hence be separated and be considered for backport.